docs/preview/features/powershell/azure-table-storage.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Scripts related to interacting with Azure Table Storage"
+layout: default
+---
+
+# Azure Table Storage
+
+## Installation
+
+To have access to the following features, you have to import the module:
+
+```powershell
+PS> Import-Module -Name Arcus.Scripting.TableStorage
+```
+
+## Create new table in Azure Table Storage on Storage Account
+
+(Re)Create a Azure Table Storage based to an Storage Account.
+
+| Parameter            | Mandatory | Description                                                                                                     |
+| -------------------- | --------- | --------------------------------------------------------------------------------------------------------------- |
+| `ResourceGroupName`  | yes       | The resource group where the Table Storage is located                                                           |
+| `StorageAccountName` | yes       | The name of the Storage Account to add the table to                                                             |
+| `TableName`          | yes       | The name of the table to add on the Storage Account                                                             |
+| `DeleteAndCreate`    | no        | The optional flag to indicate whether or not a possible already existing table should be deleted and re-created |
+
+**Example**
+
+With non-existing table:
+
+```powershell
+PS> Create-AzTableStorageAccountTable -ResourceGroupName "stock" -StorageAccountName "admin" -TableName "products"
```

Comment:
Perhaps we can rename this to: `Create-AzTableStorage`.
Or do you think that would be to trimmed down?

Reply:
Is good for me. Better even. But I wasn't sure of the 'StorageAccount' that was present in the 'old' script and if it had a meaning it was placed in the name. So I left it there.